data/json/snippets/mutant_anatomy.json

```
@@ -43,6 +43,182 @@
+    "text": [
+      {
+        "id": "cooked_mutant_brain_1",
+        "text": "This is a piece of cooked brain from a mutated animal.  Whatever gave its surface the oily sheen coagulated into bluish clumps of goo settled into the gyri.  You try to scrape them off, but they seem fused with the tissue beneath."
+      },
+      {
+        "id": "cooked_mutant_brain_2",
+        "text": "A piece of cooked brain from a heavily mutated animal.  A vein at the base of it burst while cooking, slathering the whole thing in faintly citrus-smelling yellow liquid.  While the smell would be almost appetizing, the faint taste of ammonia is anything but."
+      },
+      {
+        "id": "cooked_mutant_brain_3",
+        "text": "A piece of cooked brain from a mutant.  The eyes jutting out of the surface have grown milky, and a good few of them fell off, each revealing a small depression dominated by a short root of veins and nerve.  Judging by the spongy texture, some remain buried in the deeper tissue."
+      },
+      {
+        "id": "cooked_mutant_brain_4",
+        "text": "A cooked piece of what passes for a brain in mutant animals.  The whole thing came apart into a coarse jelly interspersed with hair-thin, branching nerves that seem move slightly.  The texture is horrible, but it tastes better than starvation."
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "mutant_lung_desc",
+    "text": [
+      {
+        "id": "mutant_lung_1",
+        "text": "A piece of lung you removed from a mutated animal.  Strange, swaying tubes filled with a clear liquid emerge from the mostly-normal lung tissue.  Some broke off during the butchery, and the escaped liquid coats your tools, hardening on contact with the air like superglue."
+      },
+      {
+        "id": "mutant_lung_2",
+        "text": "A piece of lung from a heavily mutated animal.  Even after removing the diaphragm, the tissue itself keeps rhytmically expanding and contracting, sending a gurgling whine through the windpipe with each breath.  It takes far too long for it to grow still."
+      },
+      {
+        "id": "mutant_lung_3",
+        "text": "A piece of lung from a heavily mutated animal.  The airways are broken up in regular intervals by what resemble vocal cords, and whenever you push some air through the tissue a cacophony of screams erupt from it."
+      },
+      {
+        "id": "mutant_lung_4",
+        "text": "A piece of lung from a heavily mutated animal.  The whole thing is filled with a foul-smelling, greenish foam bubbling from strange glands lined along the trachea, making you wonder how the creature could breathe at all.  It does explain the attitude, at least."
+      },
+      {
+        "id": "mutant_lung_5",
+        "text": "A piece of lung from a heavily mutated animal.  The airways are lined with rows of tiny, stone-hard barbs, each pointing towards the outside, as if they were made to keep something out."
```

```suggestion
        "text": "A piece of lung from a heavily-mutated animal.  The airways are lined with rows of tiny, stone-hard barbs, each pointing towards the outside, as if they were made to keep something out."
```
BTW, are the ids supposed to correspond to the ids when cooked somehow?

---

No, it's just for consistency's sake. I did riff on the uncooked ones for a few that sounded fun, but it's not 1:1.